tests/robustness/powerfailure_test.go

```diff
@@ -146,11 +146,11 @@ func doPowerFailure(t *testing.T, du time.Duration, fsType dmflakey.FSType, mkfs
 	dbPath := filepath.Join(root, "boltdb")
 
 	args := []string{"bbolt", "bench",
-		"-work", // keep the database
-		"-path", dbPath,
-		"-count=1000000000",
-		"-batch-size=5", // separate total count into multiple truncation
-		"-value-size=512",
+		"--work", // keep the database
+		"--path", dbPath,
+		"--count=1000000000",
+		"--batch-size=5", // separate total count into multiple truncation
+		"--value-size=512",
```

Comment:
This seems like a breaking change.

Reply:
This **is a breaking change**. That's why we were holding this migration until after releasing 1.4.0, and targeting 2.0.0.